applications/serial_lte_modem/src/lwm2m_carrier/slm_at_carrier.c

```diff
@@ -220,6 +246,13 @@ int lwm2m_carrier_event_handler(const lwm2m_carrier_event_t *event)
 
 	rsp_send("\r\n#XCARRIEREVT: %d,%d\r\n", event->type, err);
 
+#if defined(SLM_CARRIER_FOTA_AUTOMATION)
+	/* Allow time for the URC be flushed before reboot */
+	if (event->type == LWM2M_CARRIER_EVENT_REBOOT && err == 0) {
+		k_sleep(SLM_UART_RESPONSE_DELAY);
```

Comment:
Why not rather put this [here](https://github.com/nrfconnect/sdk-nrf/pull/16360/files#diff-1c276f4a29001e10696ebbf7251866a78b4593747545d5a151ca4c4e44040dd4R214)?

Reply:
No, after URC is sent